src/yarp.c

```diff
@@ -9771,7 +9776,7 @@ parse_arguments_list(yp_parser_t *parser, yp_arguments_t *arguments, bool accept
         // If we get here, then the subsequent token cannot be used as an infix
         // operator. In this case we assume the subsequent token is part of an
         // argument to this method call.
-        parse_arguments(parser, arguments, true, YP_TOKEN_EOF);
+        parse_arguments(parser, arguments, true, YP_TOKEN_EOF, false);
```

Comment:
Instead of passing this argument here, could we instead put the check for the trailing comma here? That way it wouldn't be inside `parse_arguments` at all and we could just put it where you're currently passing `false`